tilelang/jit/adapter/wrapper.py

```
@@ -703,6 +709,10 @@ def get_declaration(self, declare_kernel_code: str) -> str:
         # __global__ void __launch_bounds__(128) kernel_kernel(float* __restrict__ A) {\n
         return declare_kernel_code.split("{")[0]
 
+    def get_kernel_launch_code(self, function_name, grid_str, block_str, smem_str, call_args, cluster_dims):
```

Ruff (0.15.4)

[warning] Unused method argument: `cluster_dims`

(ARG002)